src/renderer/util/theme.js

```diff
@@ -1,18 +1,42 @@
-import { THEME_LINK_ID, COMMON_STYLE_ID, DEFAULT_CODE_FONT_FAMILY } from '../config'
+import { THEME_STYLE_ID, COMMON_STYLE_ID, DEFAULT_CODE_FONT_FAMILY, railscastsThemes } from '../config'
+import { dark, ulysses, graphite } from './themeColor'
 
 export const addThemeStyle = theme => {
-  const href = process.env.NODE_ENV !== 'production'
-    ? `./src/muya/themes/${theme}.css`
-    : `./static/themes/${theme}.css`
+  let themeStyleEle = document.querySelector(`#${THEME_STYLE_ID}`)
+  if (!themeStyleEle) {
+    themeStyleEle = document.createElement('style')
+    themeStyleEle.id = THEME_STYLE_ID
+    document.head.appendChild(themeStyleEle)
+  }
+  switch (theme) {
+    case 'light':
+      themeStyleEle.innerHTML = ''
+      break
+    case 'dark':
```

Comment:
I would rename `dark` to `material-dark` and also add `dark` in the case. I'll add another `dark` theme later:
```js
case 'dark':
case 'material-dark':
 // ...
`

Reply:
great! I'll wait for you to see a new dark theme, you can add you favourite `themeColor`, Also you can add a another light theme.

>I would rename dark to material-dark

great!